### PR TITLE
Add env config script for use in CI

### LINKED ISSUE #186: Add CI env config script
CI deployed instances should reuse a portion of the install.sh

### PR DESCRIPTION
**Pull request by copejon**
**Which issue(s) this PR addresses**:

Closes #186

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,13 +56,24 @@ sudo dnf install -y glibc-static
...
+
 ### Running
 
-Use `install.sh` to set up your sytem and install run-time dependencies for Microshift, then simply:
 ```
 sudo microshift run
 ```
 
+Developers:
+
```
Comment (fzdarsky) [inline README.md:76]: Is this necessary, given that we're already in the "Developing MicroShift" section?

File: install.sh [hunk 2]
```diff
@@ -3,8 +3,17 @@ set -e -o pipefail
...
 # Usage:
 # ./install.sh
+# set ""
 
-VERSION=$(curl -s https://api.github.com/repos/redhat-et/microshift/releases | grep tag_name | head -n 1 | cut -d '"' -f 4)
+# ENV VARS
+# MICROSHIFT_CONFIG_ENV_ONLY=true will short-circuit the installation immediately after the host is configured, and before
+# the microshift release is downloaded.  This is to allow developers and CI to configure a host envirnment for testing
+# non-release microshift runtimes.
+MICROSHIFT_CONFIG_ENV_ONLY=${MICROSHIFT_CONFIG_ENV_ONLY:=false}
```
Comment (fzdarsky) [inline install.sh:12]: Do we have a scheme for how we want to call env vars for install.sh (vs. those we pass into MicroShift)?
Just want to ensure we don't accidentally collide.

Comment (copejon) [reply install.sh:12]: Yes - the scheme is `MICROSHIFT_*`. Will drop the prefix to keep it cleanly separated.

**Review by fzdarsky — APPROVED** [review]

**Pull request merged**